_docs/glider-background-and-sampling-terminology.md

```diff
@@ -20,18 +20,18 @@ summary: This page defines glider and sampling terminology used throughout the r
+ + [Slocum](https://www.teledynemarine.com/en-us/products/product-line/Pages/Autonomous-Underwater-Glider.aspx): designed and built by [Teledyne Webb Research Corporation](https://www.teledynemarine.com/brands/webb-research/).
 
 ## Sampling Pattern Terminology
 
 The schematic and definitions below define the sampling terminology of a profiling glider.  While all of the terms defined below are commonly used in the glider community, the 2 fundamental terms used by the **NGDAC** to organize data are the **profile** and **trajectory**.  The **NGDAC** receives glider data as individual, sequentially numbered **profiles** and aggregates files from the same **trajectory** into a single data set representing the deployment.
 
-<!-- ![Glider Sampling Patterns and Terms](https://raw.githubusercontent.com/kerfoot/ioosngdac/master/doco/glider-sampling-terminology.png) -->
+<!-- ![Glider Sampling Patterns and Terms](https://raw.githubusercontent.com/kerfoot/glider-dac/master/doco/glider-sampling-terminology.png) -->
 ![Glider Sampling Patterns and Terms](/glider-dac/glider-sampling-terminology.png)
 
- + **Profile**: A single vertically oriented track of a glider, either upward or downward through the water column.  A profile is one-half of a **dive**.  The profile is the fundamental atomic data type used by the **NGDAC**.  All data submitted to the **NGDAC** is submitted as individual profiles, containing the various water column properties or sensor values.  Examples of the file format description can be found as [CDL](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl), [NetCDF file](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc) and [ncml](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml) can be found [here](https://github.com/kerfoot/ioosngdac/tree/master/nc/template).
+ + **Profile**: A single vertically oriented track of a glider, either upward or downward through the water column.  A profile is one-half of a **dive**.  The profile is the fundamental atomic data type used by the **NGDAC**.  All data submitted to the **NGDAC** is submitted as individual profiles, containing the various water column properties or sensor values.  Examples of the file format description can be found as [CDL](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl), [NetCDF file](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc) and [ncml](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml) can be found [here](https://github.com/kerfoot/glider-dac/tree/master/nc/template).
```

Comment:
These are problematic b/c they are links to John's fork. Should we transfer those files here and fix these links? Should we review/re-write this to be sure we still need this?

Reply:
Yes, transfer files and fix links.  Documents shouldn't be stored in forks.

Reply:
Done!